### ISSUE TITLE
How to set Negative and  Positive text with diff color 

### ISSUE THREAD

**Issue by EHG613** [maxkeppeler/sheets#104]
How to set Negative and  Positive text with diff color ?  Need help!

**Comment by EHG613** [comment]
<img width="471" alt="image" src="https://user-images.githubusercontent.com/12960149/153383935-92938d8a-6d01-49e0-a023-05631138085c.png">

**Comment by maxkeppeler** [comment]
It was still missing. I just added some ways to change the colors. Either through your styles.xml or for each sheet individually.

Styles:
```
    <item name="sheetsNegativeButtonColor">color<item>
    <item name="sheetsPositiveButtonColor">color<item>
```
On each Sheet:
```
positiveButtonColor()
positiveButtonColorRes()
negativeButtonColor()
negativeButtonColorRes() 
```

Will be included in the next version 2.2.7
